data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1228 @@
+			`	However, as you fly around, you notice a small Gegno shuttle parked on the ground. The tan coloration of its hull suggests that it's a civilian craft, but what it's doing here is unknown.`
+			choice
+				`	(Investigate.)`
+					goto investigate
+				`	(Leave them be.)`
+					decline
+
+			label investigate
+			action
+				set "gegno: approached bystander"
+			`	You steer your ship towards the shuttle, and once you approach it, you are able to see a Gegno on the ground fiddling with one of the nearby devices. Though, as you fly closer, they stop their work and return to their shuttle, taking off shortly after. It seems like they weren't interested in any potential interaction with you.`
+			`	The satellite they were tinkering with doesn't strike you as anything special, being just another antique device amongst the others.`
+				decline
+
+
+mission "Scin on Cyife"
+	invisible
+	landing
+	source "Cyife"
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on offer
+		"gegno: encounters" ++
+		conversation
+			`Nothing is of much interest on a cold, unwelcoming moon in the middle of nowhere. However, there's something about the surface of Cyife that is a bit peculiar. Some of the light refracted through the surface seemingly dissipates or reflects back in strange patterns. Although it appears to be ice, more detailed inspection reveals that small patches of the terrain seem to instead be made of some other type of crystalline material. In the distance is something that truly stands out on this world - a large egg-shaped structure with a green hue to it.`
+
+			branch "boarded epidote"
+				has "scin: entered epidote"
+
+			branch "saw epidote"
+				has "scin: approached epidote"
+
+			`	Once you get close enough, the oblong object reveals itself to be some sort of large Gegno ship with the coloring of the Scin faction. What it's doing this far away from their home space is a mystery, much like the moon it's stationed on. After a few seconds, it launches into the atmosphere and leaves the moon. There doesn't seem to be anything left where it was parked, nor any trace of what the Scin were doing here to begin with.`
+				decline
+
+			label "saw epidote"
+			`	Once you get close enough, the oblong object is revealed to be a large Gegno Scin ship, similar to the one you encountered during an interaction with the Scin a while ago. What it's doing this far away from their home space is a mystery, much like the moon it's stationed on. Or at least, was stationed on, as after a few seconds it launches into the atmosphere and leaves the moon. Perhaps it was some sort of surveillance ship spying on the nearby Gegno Vi systems.`
```

Comment:
Having `first` conjecture (correctly, I assume) that it's a surveillance ship seems like a bit of a stretch with the information given, imo. What do you think?